docs/modules/kafka/pages/usage-guide/security.adoc

```diff
@@ -155,6 +155,14 @@ spec:
 
 NOTE: When Kerberos is enabled it is also required to enable TLS for maximum security.
 
+==== Clients
+
+In order to keep client configuration as uncluttered as possible, each kerberized Kafka broker has two principals: one for the broker itself and one for the bootstrap service.
+The client can connect to the bootstrap service, which returns the broker quorum for use in subsequent operations.
+This is transparent as each connection dynamically uses the relevant principal (broker or bootstrap).
+In order for this to work it is necessary for kerberized clusters to define an extra kafka listener for the bootstrap, with a corresponding service (and port).
+The bootstrap address is written to the discovery ConfigMap, using the Stackable bootstrap listener with the port being either 9094 (non-secure) or 9095 (secure) for kerberized clusters, and 9092 (non-secure) or 9093 (secure) for non-kerberized ones.
```

Comment:
Port 9094 is currently not used. Should we mention this? But I am also okay with the text as it is written now.

Reply:
I've clarified it a bit: 74f2cb8